tests/fixtures/.secrets

```diff
@@ -0,0 +1,3 @@
+AZURE_CLIENT_ID="somevalue"
+AZURE_CLIENT_SECRET="somesecret"
+TEAM_IDS='["demo"]'
```

Comment:
I am confused why TEAM_IDS is considered a secret.

Reply:
Does it matter if you consider it so? It is a transport mechanism in use to configure testing env. And I use it as such.